src/main.ts

```diff
@@ -3,3 +3,11 @@
 export function version(): string {
     return '1.0';
 }
+
+/*
+    Protocol-common version of a sleep function
+    input is milliseconds
+*/
+export async function delay(ms: number): Promise<any> {
```

Comment:
seemed like over kill to create a static class for this method and it makes consumers code less bulky.  I can change it if thats better.